src/controllers/userController.ts

```diff
@@ -38,4 +40,18 @@ export const userController = {
 
     return c.json({ data: deletedUser, ok: true }, 200);
   },
+
+  async like(c: IdContext) {
+    const { id } = c.req.valid('param');
+    const comment = await commentService.like(id);
+
```

Comment:
did you add logic for checking which user liked comment. one user cant like and dislike comment. so if user once liked comment and after some time he suggest to change like into dislike like from this comment should be decreased

Reply:
fixed now